packages/assets/scss/03-organisms/_table-of-contents-hierarchy.scss

```
@@ -55,7 +55,7 @@
       a {
         border-bottom-width: 3px;
         border-bottom-style: solid;
-        border-bottom-color: rgba(20, 85, 143, 0.5);
+        border-bottom-color: rgba($c-bay-blue, 0.5);
```

```suggestion
        border-bottom-color: $c-primary-lighter;
```
I think we generally want to reference the mapped theme colors

---

Done.